crates/orderbook/src/arguments.rs

```diff
@@ -44,6 +44,14 @@ pub struct Arguments {
     )]
     pub max_order_validity_period: Duration,
 
+    /// The time period an EIP1271-quote request is valid.
+    #[clap(long, env, default_value = "600")]
+    pub eip1271_quote_validity_seconds: i64,
```

Comment:
I agree it should be Duration. The `_seconds` postfix still makes sense because we need to use a manual parsing function `parse(try_from_str = shared::arguments::duration_from_seconds)` for the argument. From the perspective of the code indicating the unit is not needed but from the perspective of the person setting the argument it can help.